spec/system/planning_applications/consulting/send_letters_to_neighbours_spec.rb

```diff
@@ -367,11 +364,8 @@
         fill_in("Resend reason",
           with: "Previous letter mistakenly listed applicant's address as Buckingham Palace.")
 
-        travel_to(Time.zone.now.change(hour: 16)) do
-          expect_any_instance_of(Notifications::Client).to receive(:send_letter).with(template_id: anything,
-            personalisation: hash_including(message: match_regex(/# Application updated\nThis application has been updated. Reason: Previous letter mistakenly listed applicant's address as Buckingham Palace.\n\n# Submit your comments by #{(1.business_day.from_now + 21.days).to_date.to_fs}\n\nDear Resident/))).and_call_original
-        end
-
+        expect_any_instance_of(Notifications::Client).to receive(:send_letter).with(template_id: anything,
```

Comment:
I generally avoid `expect_any_instance_of` as it sometimes has unintended side effects where there could be multiple instances of a class, e.g. an Active Record model but it's probably okay here since there's unlikely to be a second instance and stubbing `new` on `Notifications::Client` seems to be just going through the motions for no real benefit.